web/conf/routes

```diff
@@ -78,6 +78,12 @@ POST        /u/groups/:group/partners/:partner/venues/create
 GET         /u/groups/:group/partners/:partner/venues/:venue                    fr.gospeak.web.pages.orga.partners.venues.VenueCtrl.detail(group: Group.Slug, partner: Partner.Slug, venue: Venue.Id)
 GET         /u/groups/:group/partners/:partner/venues/:venue/edit               fr.gospeak.web.pages.orga.partners.venues.VenueCtrl.edit(group: Group.Slug, partner: Partner.Slug, venue: Venue.Id)
 POST        /u/groups/:group/partners/:partner/venues/:venue/edit               fr.gospeak.web.pages.orga.partners.venues.VenueCtrl.doEdit(group: Group.Slug, partner: Partner.Slug, venue: Venue.Id)
+GET         /u/groups/:group/partners/:partner/contacts/create                  fr.gospeak.web.pages.orga.partners.contacts.ContactCtrl.create(group: Group.Slug, partner: Partner.Slug)
+POST        /u/groups/:group/partners/:partner/contacts/create                  fr.gospeak.web.pages.orga.partners.contacts.ContactCtrl.doCreate(group: Group.Slug, partner: Partner.Slug)
+GET         /u/groups/:group/partners/:partner/contacts/:contact                fr.gospeak.web.pages.orga.partners.contacts.ContactCtrl.detail(group: Group.Slug, partner: Partner.Slug, contact: Contact.Id)
+GET         /u/groups/:group/partners/:partner/contacts/:contact/edit           fr.gospeak.web.pages.orga.partners.contacts.ContactCtrl.edit(group: Group.Slug, partner: Partner.Slug, contact: Contact.Id)
+POST        /u/groups/:group/partners/:partner/contacts/:contact/edit           fr.gospeak.web.pages.orga.partners.contacts.ContactCtrl.doEdit(group: Group.Slug, partner: Partner.Slug, contact: Contact.Id)
+GET         /u/groups/:group/partners/:partner/contacts                         fr.gospeak.web.pages.orga.partners.contacts.ContactCtrl.list(group: Group.Slug, partner: Partner.Slug, params: Page.Params ?= Page.Params.defaults)
```

Comment:
it should be at the beginning of the 'contact' routes (to be coherent with the others).
BTW: not sure it's really needed as the `PartnerCtrl` can have a paginated list of contacts